packages/desktop-client/src/components/accounts/Account.js

```diff
@@ -480,6 +482,14 @@ function SelectedTransactionsButton({
     };
   }, [selectedItems]);
 
+  let ambiguousDuplication = useMemo(() => {
+    let transactions = [...selectedItems].map(id => getTransaction(id));
+
+    return (
+      transactions.some(t => t.is_parent) && transactions.some(t => t.is_child)
+    );
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏after your last removal for the child-transaction handling - we should block duplication if children transaction is selected IMO
```suggestion
    return (
      transactions.some(t => t.is_child)
    );
```

Reply:
Yep, think that's reasonable!